low-level/sspl_ll_d

```diff
@@ -382,8 +387,25 @@ def main(conf_reader, systemd_support):
                 or product.upper() == "LR2":
                 resume = True
 
-            thread = Thread(target=_run_thread_capture_errors,
-                            args=(curr_module, sspl_threaded_modules, msgQlist, conf_reader, product, resume))
+            # If recovery count is set, then sensor will try to recover from
+            # failure until the maximum recovery attempt. If not recoverable
+            # sensor will be made shutdown and its failure msg will be sent.
```

Comment:
\ # sensor will be shutdown and and its failure alert will be raised with its monitoring impact.

Reply:
Done.